bbsr/sct-tests/TCG2Protocol/BlackBoxTest/TCG2ProtocolBBTestConformance.c

```diff
@@ -271,50 +280,114 @@ BBTestGetCapabilityConformanceTestCheckpoint2 (
   char StructureVersionMinor;
   char ProtocolVersionMajor;
   char ProtocolVersionMinor;
-
   EFI_TCG2_BOOT_SERVICE_CAPABILITY      BootServiceCap;
-  BootServiceCap.Size = sizeof(UINT8) + (sizeof(EFI_TCG2_VERSION) * 2);
+
+  BootServiceCap.Size = sizeof(EFI_TCG2_BOOT_SERVICE_CAPABILITY);
 
   Status = TCG2->GetCapability (
                            TCG2,
                            &BootServiceCap);
 
   AssertionType = EFI_TEST_ASSERTION_PASSED;
 
+  if (Status != EFI_SUCCESS) {
+    StandardLib->RecordMessage (
+                     StandardLib,
+                     EFI_VERBOSE_LEVEL_DEFAULT,
+                     L"\r\nTCG2 Protocol GetCapability Test: GetCapability should return EFI_SUCCESS"
+                     );
+
+    AssertionType = EFI_TEST_ASSERTION_FAILED;
+  }
+
   StructureVersionMajor = BootServiceCap.StructureVersion.Major;
   StructureVersionMinor = BootServiceCap.StructureVersion.Minor;
 
-  // If the input ProtocolCapability.Size < sizeof(EFI_TCG2_BOOT_SERVICE_CAPABILITY)
-  // the function will initialize the fields included in ProtocolCapability.Size.
-
+  // TCG EFI Protocol spec 6.4.4 #4
   if ((StructureVersionMajor != 1) | (StructureVersionMinor != 1)) {
-     StandardLib->RecordMessage (
+    StandardLib->RecordMessage (
                      StandardLib,
                      EFI_VERBOSE_LEVEL_DEFAULT,
-                     L"\r\nTCG2 Protocol GetCapablity Test: Unexpected struct version numbers returned"
+                     L"\r\nTCG2 Protocol GetCapability Test: GetCapability should have StructureVersion 1.1"
```

Comment:
Can we print out the values of StructureVersionMajor and StructureVersionMinor here to make it easier to debug.  It would be helpful to know the actual values returned by the GetCapability API.

Reply:
Added debug prints for ProtocolVersion and StructureVersion.